frontend/src/services/record-audio/record-audio.service.ts

```diff
@@ -28,7 +27,7 @@ class RecordAudio {
   }
 
   public start(): void {
-    this.#recorder?.start();
+    this.#recorder?.start(500);
```

Comment:
What does it meen 500?

Reply:
This parameter takes a value in milliseconds, and overrides the length of the media capture chunk to be returned in each Blob.

Reply:
Magic number? it should be constant?

Reply:
You right. It should be constant